### ISSUE TITLE
[v2] Email not set in git config causes crash

### ISSUE THREAD

**Issue by TechplexEngineer** [wailsapp/wails#730]
**Description**
I choose not to have my email address in my ~/.gitconfig and instead choose to set the email on a per-repository basis. [git.Email()](https://github.com/wailsapp/wails/blob/d5cbfa67490b223c76bd148a3db6c781e51037d4/v2/pkg/git/git.go#L24) seems to run the command `git config user.email` which exits with exit code 1 which the programs erroneously treats as an error.

**To Reproduce**
1. `git clone https://github.com/wailsapp/wails.git`
2. `cd wails`
3. `git checkout v2-alpha`
4. `cd v2/cmd/wails`
5. `go build`
6. `wails init -n test`

Command Output:
```
Wails CLI v2.0.0-alpha.65

Initialising Project Test
-------------------------


ERROR: exit status 1
```

**Expected behaviour**
Give an error stating that having email in git config is required, or make the failure of git.Email() an allowable failure

**System Details**
Linux Mint 20 Ulyana (based on Ubuntu 20.04)
go version go1.16.2 linux/amd64

**Comment by leaanthony** [comment]
Hey thanks for testing v2! This should be allowable, it's only to use in the Author field of generated code.

**Comment by leaanthony** [comment]
This should be fixed in `v2.0.0-alpha.66`. Please let me know how you get on 👍
